### DISCUSSION TITLE
crsf+frsky for telemetry at the same time

### DISCUSSION THREAD

**Discussion by secretman28** [EdgeTX/edgetx#930] [Ideas]
im really please of edgetx nothing to say really love it ,  i got a question/request, i know i can use both the internal and external module at the same time, its test and working, but im flying turbine jet, nothing have crsf telemetry support, i try to get telemetry over a s.port from an rxs-r and control over crsf, i can bind both both receive signal but i cant enter the setting in crossfire(agent lite or the old lua for crsf find no module), but rc work, and the frsky rxs-r send telemetry but on off on off loos telemetry and recover etc... did we have a way to do this' i think its because both receive info collide in the uart, i dont really know but adding just a option like the disable telemetry on frsky or dsm to crsf is doable ??? or add something to combine both telemetry ?

i really dont want to put frsky in the jet, crsf prove me that is bullet proof rc link and want to keep it my favorite
i know its a lot of work or thinking to make this work but sometime its really easy if you dont ask you cannot know

**Comment by secretman28** [comment]
one way i found to make it work is crsf in ppm and frsky normale, but cant play with crsf setting or see the lq on screen

**Reply by raphaelcoeffic** [reply]
Hey! Which radio is that? If you want to use both at the same time, it won’t work with iXJT/XJT FrSky modules, as they grab the S.PORT for telemetry. It would however with internal MPM and CRSF module, or with ISRM (some mod required to cut the S.PORT line from the ISRM).

**Comment by secretman28** [comment]
radio master internal mpm and ext crsf, both not work both telemetry collaps canot find crsf in crsf lua and frsky telemetry lost back lost back... canot get any telemetry discover if i put ppm for external.module crsf frsky telemetry work

**Reply by raphaelcoeffic** [reply]
Oh, I see! So the ideal solution would be to fix the issues raised when both modules use telemetry:
- Internal MPM using D16.
- External CRSF using telemetry for LUA.

It seems to me these are 2 separate issues:
1. Internal MPM & external CRSF conflicting with telemetry
2. LUA `crossfireTelemetryPop` / `crossfireTelemetryPush` conflicting with internal MPM

**Reply by raphaelcoeffic** [reply]
@secretman28 I created this issue to document / propose a change: https://github.com/EdgeTX/edgetx/issues/944

**Reply by secretman28** [reply]
tanks it can be usefull to other pepole too realy apreciate
